go/vt/mysqlctl/mysqld.go

```diff
@@ -87,15 +93,15 @@ var (
 
 	replicationConnectRetry = 10 * time.Second
 
-	versionRegex = regexp.MustCompile(`Ver ([0-9]+)\.([0-9]+)\.([0-9]+)`)
+	versionRegex = regexp.MustCompile(fmt.Sprintf(`%s([0-9]+)\.([0-9]+)\.([0-9]+)`, versionStringPrefix))
```

Comment:
I'm sure there's a good reason you changed this, but it doesn't feel necessary.

Reply:
Just so that we're using the same const in the regex and where we build the string to match it when querying mysqld.